src/parse.d

```diff
@@ -3049,16 +3049,14 @@ public:
      */
     Dsymbol parseAggregate()
     {
-        AggregateDeclaration a = null;
-        int anon = 0;
-        Identifier id;
         TemplateParameters* tpl = null;
-        Expression constraint = null;
+        Expression constraint;
```

Comment:
Can you move this variable just before the corresponding if - `if (token.value == TOKif)`?